contracts/voting-verifier/src/contract.rs

```diff
@@ -425,6 +453,12 @@ mod test {
                 address_format: AddressFormat::Sui,
                 should_fail: true,
             },
+            TestCase {
+                source_gateway_address:
+                    "aleo1q3t7cjwk9ncxcdxfm8r5ax83mzudd923gffncv5egfjyevfevuyscvcvz".to_string(),
```

Comment:
ok to hardcode?

Reply:
Thanks for spotting it. It's ok to be hardcoded here because it is the unit test setup. It is the same above for other chains.